tests/lib/collectors/utils/requester.ts

```diff
@@ -17,7 +17,7 @@ const text = `This is a text
 
 test.beforeEach(async (t) => {
     const server = createServer();
-    const requester = new Requester();
+    const requester = new Requester({});
```

Comment:
@molant Done. I changed the requester constructor so we don't need the empty object.